internal/manifests/opampbridge/annotations.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opampbridge
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
+	"github.com/open-telemetry/opentelemetry-operator/internal/manifests/manifestutils"
+)
+
+const configMapHashAnnotationKey = "opentelemetry-opampbridge-config/hash"
+
+// Annotations returns the annotations for the OPAmpBridge Pod.
+func Annotations(instance v1alpha1.OpAMPBridge, configMap *v1.ConfigMap, filterAnnotations []string) map[string]string {
```

Comment:
i wonder if we should also move annotations.go and labels.go to a shared thing in manifestutils?

Reply:
yeah that makes sense, want to open an issue about that?